etna/transforms/decomposition/change_points.py

```diff
@@ -0,0 +1,72 @@
+from typing import List
+from typing import Optional
+from typing import Tuple
+from typing import Type
+
+import pandas as pd
+from ruptures.base import BaseEstimator
+from sklearn.base import RegressorMixin
+
+from etna.analysis.change_points_trend.search import _find_change_points_segment
+from etna.transforms.base import Transform
+
+TTimestampInterval = Tuple[pd.Timestamp, pd.Timestamp]
+TDetrendModel = Type[RegressorMixin]
+
+
+class ChangePointsTransform(Transform):
```

Comment:
`ChangePointsTransform` should be "private" because it works with the only segment, shouldn't it?